### ISSUE TITLE
decision_sender_create function definition is dead code

### ISSUE THREAD

**Issue by SchuylerGoodman** [VowpalWabbit/reinforcement_learning#208]
It looks like this line is dead code, since DECISION_SENDER was removed in commit #189 (@olgavrou ?)

https://github.com/VowpalWabbit/reinforcement_learning/blob/6f4600f01c55bb410e6a8f988801655580ccd124/rlclientlib/azure_factories.cc#L16

**Comment by olgavrou** [comment]
@SchuylerGoodman yes it looks like that should be removed also, thanks for pointing it out
